(window.webpackJsonp=window.webpackJsonp||[]).push([[16],{424:function(e,t,o){var content=o(465);"string"==typeof content&&(content=[[e.i,content,""]]),content.locals&&(e.exports=content.locals);(0,o(18).default)("4281134a",content,!0,{sourceMap:!1})},464:function(e,t,o){"use strict";var n=o(424);o.n(n).a},465:function(e,t,o){(t=o(17)(!1)).push([e.i,".p_indent[data-v-634cd977]{font-size:24px;text-indent:50px}.fig_title[data-v-634cd977]{text-align:center}.h2_text[data-v-634cd977]{font-size:30px}.link_style[data-v-634cd977],.link_style[data-v-634cd977]:active{color:#507ea4}.link_style[data-v-634cd977]:link,.link_style[data-v-634cd977]:visited{color:#507ea4;text-decoration:underline}.last_message[data-v-634cd977]{margin:400px}.left_box[data-v-634cd977]{padding:0}.sidebar[data-v-634cd977]{height:100vh;position:fixed;width:17%;background-image:url(https://2020.igem.org/wiki/images/e/e9/T--Waseda--Tasukinisakura1.png);background-size:100% auto;background-repeat:repeat}.main_text[data-v-634cd977]{padding:40px 10%;background:#fff 0 0 no-repeat padding-box}.h1_box[data-v-634cd977]{border-bottom:5px solid #000;margin-bottom:10px}.h1_box[data-v-634cd977],.h1_box_text[data-v-634cd977]{padding-bottom:0}.h1_text[data-v-634cd977]{font-size:60px}.h1_imbox[data-v-634cd977]{display:block;position:relative}.h1_image[data-v-634cd977]{position:absolute;top:-30px;left:-30px;height:150px}h2[data-v-634cd977]{font-size:30px;border-bottom:5px solid #000;margin:20px 0}.image_top[data-v-634cd977]{height:0;padding-top:66%;background-size:contain;background-image:url(https://2020.igem.org/wiki/images/6/63/T--Waseda--AttributionsTop.png)}.navi_icon[data-v-634cd977]{width:200px;height:200px}.navi_image[data-v-634cd977]{position:relative;padding-top:20px;margin:0 auto}.navi_image[data-v-634cd977],.navi_text[data-v-634cd977]{display:block;width:200px}.navi_text[data-v-634cd977]{margin:-40px auto 0;padding:60px 0 40px;border-radius:20px;background-color:#fff}.navi_svg[data-v-634cd977]{position:absolute;top:40px;left:20px}#triangle[data-v-634cd977],.navi_svg[data-v-634cd977]{width:160px;height:160px}#triangle[data-v-634cd977]{stroke-dasharray:510.293}.title_text[data-v-634cd977]{display:block;margin:10px auto;width:60%;border-radius:10px;background-color:None}.title_text_active[data-v-634cd977]{background-color:#507ea4;color:#fff;opacity:.5}.row_max-box[data-v-634cd977]{width:100%;padding:0;margin:0}.last_message[data-v-634cd977]{margin-bottom:400px}a[data-v-634cd977]:link{text-decoration:none}a[data-v-634cd977]:link,a[data-v-634cd977]:visited{color:#000}a[data-v-634cd977]:hover{color:#ff69b4}a[data-v-634cd977]:active{color:#000}",""]),e.exports=t},511:function(e,t,o){"use strict";o.r(t);o(7);var n={data:function(){return{fab:!1,scrollY:0,scrollPer:510.293,sectionOffsetTop:[],check:[!1,!1,!1,!1,!1,!1,!1,!1]}},methods:{pushScrollY:function(e){this.scrollY=window.scrollY},pushElementOffsetTop:function(){var e=this;["WetLab","Modeling","NaviVideo","Wiki","HP","NaviOther","NaviLast"].forEach((function(t){var element=document.getElementById(t),o=Math.round(window.scrollY+element.getBoundingClientRect().top);e.sectionOffsetTop.push(o)}))}}},r=(o(464),o(86)),l=o(124),c=o.n(l),m=o(396),d=o(397),h=o(401),f=o(189),_=o.n(f),v=o(125),component=Object(r.a)(n,(function(){var e=this,t=e.$createElement,o=e._self._c||t;return o("v-container",{attrs:{fluid:"","ma-0":"","pa-0":"","fill-height":""}},[o("v-container",{attrs:{fluid:"","ma-0":"","pa-0":""}},[o("v-row",{attrs:{fluid:"","ma-5":"","pa-5":"","mb-2":"",justify:"center"}},[e.$vuetify.breakpoint.lg||e.$vuetify.breakpoint.xl?o("v-col",{staticClass:"left_box",attrs:{align:"left"}},[o("v-col",{staticClass:"sidebar",attrs:{"ma-0":"","pa-0":""}},[o("v-row",{staticClass:"navi_image"},[o("svg",{staticClass:"navi_icon",attrs:{"z-index":"99"}},[o("circle",{attrs:{fill:"#363636",cx:"100",cy:"100",r:"80",stroke:"#ffffff","stroke-width":"18"}}),e._v(" "),o("circle",{style:{"stroke-dashoffset":e.scrollPer},attrs:{fill:"none",cx:"100",cy:"100",r:"80",stroke:"#008080","stroke-width":"18",id:"triangle",transform:"rotate(270 100 100)"}}),e._v(" "),e._v("\n                                \n                                Sorry, your browser does not support inline SVG. \n                              ")]),e._v(" "),o("img",{staticClass:"navi_svg",attrs:{"z-indx":"10",src:"https://2020.igem.org/wiki/images/6/6a/T--Waseda--iGEMprojectIcon.svg"}})]),e._v(" "),o("v-row",{staticClass:"navi_text"})],1)],1):e._e(),e._v(" "),o("v-col",{staticClass:"main_text",attrs:{cols:"12",lg:"10",xl:"10"}},[o("v-row",{directives:[{name:"scroll",rawName:"v-scroll",value:e.pushScrollY,expression:"pushScrollY"}],attrs:{id:"Monoterpene"}},[o("v-row",{staticClass:"row_max-box"},[o("h2",{staticClass:"h2_text"},[e._v("zombie model")])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            In this COVID-19 crisis, we, iGEM Waseda, tried to attract high school students and general public to synthetic biology through three tactics.   First, since cell-free system is relatively easy to operate, we wanted to provide an opportunity high school students and general public to experience synthetic biology hands-on. We thus proceeded with our iGEM project adopting the cell-free system. This was an advantage for us to continue with our wet lab experiment in this COVID-19 pandemic situation. Second, we have developed a comprehensible tool and given an interesting story to modeling. Modeling plays an essential role in synthetic biology, but it is difficult to explain accurately to the general public. These stories and tools will help them to understand and enjoy synthetic biology. Last but not least, the theme we are dealing with must not only be interesting as a story, but also biologically. This year, we thus decided to study decision making by the toggle switch gene circuit, which is an important basis in developmental biology and regenerative medicine; in those research fields, cell-fate decision making is quite an important notion.     \n                                                  \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/c/ca/T--Waseda--cellfreebenefit_illustration.png",alt:"Fig.2-1-1 Benefit of cell-free system "}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-1 Benefit of cell-free system ")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/a/a5/T--Waseda--zombievssamuraimovie.png",alt:"Fig.2-1-2 Image ou our project"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-2 Image ou our projec")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/e/ed/T--Waseda--waddingtonlandscape.png",alt:"Fig.2-1-3 Image of landscape of cell fate"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-3 Image of landscape of cell fate")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            The story we developed was the war between zombies and Samurai. Both Zombie and Samurai are essentially human beings, but their states switch by some factor  (Fig.2-1-4 top).  Zombies are able to make Samurai join their group by releasing communication molecules (Fig.2-1-4 middle).  On the other hand, Samurai also try to save the companion.  At the end of the battle, the winner side is defined by the concentration of communication molecules in the field (Fig.2-1-4 bottom).       \n\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/3/38/T--Waseda--stateswitchwithfactor.png",alt:"Fig.2-1-4 State switching with factors"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-4 State switching with factors")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/6/6d/T--Waseda--SamuraiandZombieanime3.gif",alt:"Fig.2-1-5 our genetic circuit"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-5 our genetic circuit")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \n                            A genetic circuit for the Zombies and Samurai war (or cell-fate decision system) consists of two sub-systems: toggle switch and cell-cell communication module.  The two states, Samurai and Zombie, are achieved by two stable states of a toggle switch circuit composed of two repressors each of which inhibiting gene expression of the other (Fig.2-1-6 left).   A Zombie-state cell produces not only Zombie repressor but also Zombie communication molecule produced by an enzyme encoded downstream of the repressor. The increase of the Zombie cells increase the concentration of Zombie communication molecule designed to convert a Samurai cell.  For that conversion mechanism, all of the cells, even a cell in a Samurai state has Zombie repressor gene regulated by Zombie communication molecule.  For the symmetry of the scenario, all of the cells have genes for Samurai repressor and enzyme producing Samurai communication molecule. \n\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/4/4e/T--Waseda--idealresult4.png ",alt:"Fig.2-1-6 The ideal results of our gene circuit"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-6 The ideal results of our gene circuit")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \n                              To demonstrate this model, we solved the ordinary differential equations containing Hill repression and activation terms.  The ideal result for the war between Samurai and Zombie is in the right panel of Fig.2-1-6 If there are more samurai cells in the initial state, almost all cells will be samurai at the end point, and vice versa.  This has previously been achieved with ordinary toggle switch genetic circuits without cell-cell communication. \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \nBefore modeling of in vitro systems with cell-cell communication, we established differential-equation based modeling for the same circuit which works in living "),o("i",[e._v("E. Coli")]),e._v(" cells (Fig.2-1-7) because of accumulated cases for such in vivo models. In this modeling, we assumed the behavior of E. coli in vivo, and used parameters adjusted somewhat to behave as designed.  From the appearance of the nullcline (Fig.2-1-7), the equilibrium points are arranged in the same way as a normal toggle switch.  Thus, once our parameter search found the adjusted parameters for cell-free systems, we were able to realize bistable system. \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/8/86/T--Waseda--equationandnull2.png",alt:"Fig.2-1-7 the equation of our modeling and nullcline"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-7 the equation of our modeling and nullcline")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            Fig.2-1-8 illustrates the behavior of our gene circuit. When the zombie state cell is induced with a samurai signal, the cell fate will change depending on the size of the signal. If the zombie receives a large number of samurai signals, the zombie will not be able to withstand the signal and will turn into a samurai. However, if samurai signal received by the zombie state cell is not enough, the zombie can revert back to being a zombie again, without completely transforming into a samurai. As a result, we were able to create two final states simply by changing the initial concentration of communication molecules.                             \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/a/ac/T--Waseda--behaviornullandtime2.png",alt:"Fig.2-1-8 The concept behavior of our gene circuit "}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-8 The concept behavior of our gene circuit ")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \n                              In contrast to cellular system, a cell-free system lacks cell growth and has much smaller degradation- dilution term, which lead to longer operating time.  By using parameters from reference (Maurizi MR,1992), we thus estimated operating time for state-switch between zombie and samurai in our cell-free system. \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/c/c9/T--Waseda--degradationandpicture.png",alt:"Fig.2-1-9 Degradation ratio and time of transition between equilibrium points"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-9 Degradation ratio and time of transition between equilibrium points")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("    \n                            Fig.2-1-9 shows the the relationship between this degradation rate and operation time for state switching. The blue point is about normal protein degradation, and the green point shows the case of the LVA degradation tag are introduced on the protein. Considering a system based only on normal protein degradation（1%/hour =1.6*10^-4/min ）of a cell-free system without growth of volume, it takes too much time (approximatery 2x104 min = 300 hours) to operate a state-switch between Samurai and Zombie.  To solve this problem, we first investigated the size of the required degradation term.  Operation time for the state switch was estimated from the time that high concentration of Samurai repressors at one equilibrium point are reduced by degradation to the repressor concentration at another equilibrium point. Although we did not consider the term of protein synthesis in this calculation, there is no problem because this assumption without protein synthesis shows shorter operation time than a case with protein synthesis                          \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            Although cell-free system is selected as our project platform for implementation of the zombie vs. samurai scenario using a toggle switch circuit, cell-growth rate is known to be important for in vivo toggle switch. Its behavior can be predicted by a mathematical modeling which include a dilution-dependent decrease of concentration of repressors. For cell-free system, our modeling described above indeed have shown that cell-free toggle switch will not work if the system has low degradation rate.     \n\n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v(" \n                             In order to increase the degradation rate for specific proteins in our cell-free system, we decided to add LVA degradation tag to the repressor proteins. LVA degradation tag is a 9~11 amino acids sequence attached to the end of a protein. The tagged proteins are well degraded by a certain combination of proteins: clpX, clpP and sspB                               \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \nTo see the improvement in degradation effect by LVA degradation tagging, we constructed an improved part (BBa_K3580003) by a modification of an existing part: Plux/tet-GFP(BBa_K934025). Reduction of the concentration of proteins by both degradation/ dilution and increase of the concentration by production play an important role in the behavior of gene circuit. Although protein concentration decrease by dilution caused by the growth of the cell, cell-free systems can’t because its volume is fixed. Therefore, an improvement is necessary to incorporate degradation into the cell-free system. \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("   \n                            In order to compare in vivo activity of these parts (BBa_K3580003 and BBa_K934025), we first measured the fluorescence of GFP 240 minutes after the start of induction. The fluorescence of tagged GFP (BBa_K3580003) was lower than that of normal GFP (BBa_K934025) at 240 min point (fig 2-1-10). Although GFP is a stable protein with a β-barrel and difficult to be degraded, this result shows that tagged GFP was successfully degraded as we planned.                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/File:T--Waseda--Plux-tet-GFPssrA-vivo-comment.png  ",alt:"Fig 2-1-10 ssrA degradation tag assay in vivo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-1-10 ssrA degradation tag assay in vivo")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            Then, we compared the fluorescence of GFP in a cell-free system which consisted of cruedextract of E.coli containing luxR protein (Fig 2-1-11). Because of the programmed degradation, the fluorescence of tagged GFP(BBa_K3580003) showed slight signal nearly equal to a negative control where neither the template GFP DNA nor the inducer AHL existed. The results show that LVA degradation tagged protein can be degraded exceptionally both "),o("i",[e._v(" in vivo")]),e._v(" and "),o("i",[e._v("in vitro")]),e._v(".  We will be able to enjoy state switching in a test tube. So we further promoted our modeling.     \n\n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/File:T--Waseda--Plux-tet-GFPssrA-vitro-comment.png",alt:"Fig 2-1-11 ssrA degradationtag assay in vitro"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-1-11 ssrA degradationtag assay in vitro")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/f/f3/T--Waseda--basicwarincellfree.png",alt:"Fig.2-1-12 Basic war in cell-free system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-1-12 Basic war in cell-free system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            Our simulation showed fate of the battle between zombies and samurais in the test tube Fig.2-1-12 Depending on initial concentration of components, we can prepare a zombie test tube and a samurai test tube which are stable state. Then, by mixing zombie and samurai cell-free solutions in various ratios, the battle started. When zombie state cell-free solution and samurai state cell-free solution were mixed at a ratio of 2:8, the mixed solution was stable in the samurai state. This means that the samurais have beaten the zombies. However, a slight change of the ratio completely turned their fate. When the zombie and samurai solutions were mixed at a ratio of 3:7, the mixed solution was stable in the zombie state and the zombies willdestroyed the Samurai.                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Cross talk assay in vitro")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("      \n                              When we construct and model the “zombie vs samurai” system, a potential problem was crosstalk, by which a promoter in a quorum sensing (QS) systems is activated unexpectedly by another type of QS components. Consequently, we some experiments to reveal which combination of QS systems have less crosstalk ability in a cell-free system.\n                          ")])])],1),e._v(" "),o("v-row",{attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/e/e5/T--Waseda--crosstalk_explain.png",alt:"Fig 2-1-13  Crosstalk in the Zombie vs Samurai system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-1-13  Crosstalk in the Zombie vs Samurai system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/d/d4/T--Waseda--crosstalk_question.PNG",alt:"Fig 2-1-14  the possible crosstalks"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-1-14  the possible crosstalks")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            Among quorum sensing systems, we put focus on rhl system because of absence for in vitro crosstalk regarding Rhl system. Our candidate for the counterpart for Rhl system is lux system because Rhl system in Pseudomonas aeruginosa is known to be acticvated by Las system. Although combination between Las and Lux is also attractive candidate, Andrew.D et., al have shown crosstalk between them. (2018, Andrew.D). Therefore, we investigated whether there is a crosstalk between the lux and rhl QS systems "),o("i",[e._v("in vitro")]),e._v(". As RhlR-responsible promoters, we used a promoter set which showed improvement in S/N ratio in vivo (Tokyo tech iGEM 2014).                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"http://parts.igem.org/File:Titech2014_parts_rhl_promoter_main_Fig1.png",alt:"2-1-15  kinds of Prhl"}}),e._v(" "),o("figcaption",{staticClass:"fig_title",attrs:{"2-1-15":"",kinds:"",of:"",Prhl:""}})])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("\n                            For the crosstalk experiments, we constructed two kinds of cell-free translation systems containing a substantial amount of R protein (luxR,rhlR). In those systems, we put signaling molecule (3OHSL-C4 or 3OHSL-C6) and target genes (plux/tet-GFP or prhl-RR-GFP), respectively. We measured the fluorescence of GFP expressed from plux/tet-GFP and prhl(RR)-GFP on a RT-qPCR machine. By measuring the fluorescence of FITC atthe same time, we were able to quantitatively compare the fluorescence values of GFP in different runs.       \n\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/5/58/T--Waseda--crosstalk_results_rhlR.png",alt:"2-1-16 Result of crosstalk assay of the RhlR in cell-free system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("2-1-16 Result of crosstalk assay of the RhlR in cell-free system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Resultsz")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("      \n                              Firstly, we did not fined crosstalk between RhlR activator and Plux promoter Fig 2-1-16. In contrast to strong activation in the combination among 3OC4HSL , RhlR, and Prhl(RR) promoter, the other combinations showed little fluorescence.\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/9/9e/T--Waseda--crosstalk_results_luxR.png",alt:"2-1-17 Result of crosstalk assay of the luxR in cell-free system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("2-1-17 Result of crosstalk assay of the luxR in cell-free system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("   \n                            Unfortunately, however, we found Severe crosstalk from LuxR activator. As shown in two red bars, LuxR protein with OC6HSL activates not only Lux promoter but Rhl promoter. Althogh we tested another LuxR-responsible promoter, PRhlLR, it showd another cross talk from RhlR protein (三階層リンク).  This project thus rejected this promoter variant.                           \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/7/77/T--Waseda--crosstalk_summary.png",alt:" Fig 2-1-18 summarize our crosstalk analysis using cell-free system. "}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v(" Fig 2-1-18 summarize our crosstalk analysis using cell-free system. ")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \n                            Even with the identified severe crosstalks between Rhl promoter and LuxR-3OC6HSL complex, reduction of maximum activity of Rhl promoter allowed us to prepare test tubes for Samurai and Zombie. Because of the same activation by samurai signaling molecule for the both repressors, Samurai state seems to be not stable. As describe above, nullclines for our system without crosstalk showed three intersections and thus implied bistability (Fig.2-1-19 left).  By additional effect from the messy cross talk, shape of nullclines changes dramatically. Now, two null clines have only one intersection. This means loss of bistability and we cannot discriminate zombie and samurai state (Fig.2-1-19 center). \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v(" \n                            However, we could reduce CI repressor production from Rhl promoter by mutations at its DNA sequence. Even though such reduction had a risk of unitability of Zombie state, adequate modulation of maximum activity of Rhl promoter kept the stability. When we reduce the maximum expression by Rhl promoter to 1/10 compared with Lux promoter activity, phase space analysis show both stabilities of Zombie and Samurai state; the reduction did not change the number of intersections of nullclines from the case without crosstalk (Fig.2-1-19 right).  Moreover, for this model we drew Fig.2-1-20 as a time course of mixed Zombie and Samurai. We found that even if we consider the crosstalk, the two powers can compete and be biased to either side depending on their ratios. In other words,                             \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"})])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/2/21/T--Waseda--R_Pcrosswarincellfree.png",alt:" Fig.2-1-20 War in the cell-free system containing crosstalk between R proteins and Promotes"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v(" Fig.2-1-20 War in the cell-free system containing crosstalk between R proteins and Promotes")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",[e._v("      \n                          [1] Maurizi MR. Proteases and protein degradation in Escherichia coli. Experientia. 1992 Feb 15 48(2):178-201"),o("br"),e._v("\n                          [2] Kigawa, T., Yabuki, T., Matsuda, N. et al. Preparation of Escherichia coli cell extract for highly productive cell-free protein expression. J Struct Func Genom 5, 63–68 (2004)."),o("br")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box"},[o("h2",{staticClass:"h2_text"},[e._v("SAMURAI System")])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/a/af/T--Waseda--SAMURAI_overview.png",alt:"Fig2-3-1 Overview of SAMURAI system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig2-3-1 Overview of SAMURAI system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            The smells we have synthesized is not only for Zombie and Samurai in the story but for us in real-world. From initial stage of our project, our team has thought that synthetic biology can attract general public if we establish an interesting system. Combined with our synthesis of limonene and sabinene, the incoherent feedforward loop (IFFL) system allowed us to design “Switching And Modulated Utilization of Refreshing Aroma Integration system”,SAMURAI system, where pulse expression of each of the two monoterpene synthases switches time-dependently.     \n                                                  \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("IFFL ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("   \n                            in order to change the scents with a time lag, we designed fragrance production regulated by  IFFL(Incoherent Feed Forword loop), which can generate pulse activation of a gene. A typical IFFL consists of an activator protein  (X), and a repressor protein (Y) ,and a taget gene (Z).   \n\n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("   \n                             A target gene Z is directly activated by protein X.  Aditionally, gene Z is indirectly repressed by protein X due to accumulation of repressor Y produced by X.  Because this direct inhibition requres longer time than direct activtion, the time course of Z production shows a pulse                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"",alt:"Fix 2-3-2 Explanation of incoherent feed forward loop"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-2 Explanation of incoherent feed forward loop")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/7/7e/T--Waseda--pulse-explanation.png",alt:"Fig2-3-3 Explanation of some point of the concentration pulse"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig2-3-3 Explanation of some point of the concentration pulse")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            For time-dependent change of fragrances, we designed an expanded IFFL system containing two target genes (Z1,Z2) each of which has different peak time to the other.　By adjusting the parameters for each activation and repression, we can provide a time difference in the generation of each pulse.                              \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/b/b3/T--Waseda--Muitl_IFFL.png",alt:"Fig2-3-4 Multi IFFL gene circuit"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig2-3-4 Multi IFFL gene circuit")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/c/c2/T--Waseda--IFFL_ideal.png",alt:"Fig 2-3-5 ideal behavier of multi IFFL"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-5 ideal behavier of multi IFFL")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            By using that system, we can change fragrance automatically as time passed                             \n                          ")])]),e._v(" "),o("v-col",[o("h3"),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("\n                             In order to check whether the designed multi-target IFFL can really generate two pulse with a time delay, we extensively searched parameter space for the system                             \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/File:T--Waseda--IFFL_independent_search_kxandky.png",alt:"Fix 2-3-6 Independent param search of Kx(left) and Ky(right)"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-6 Independent param search of Kx(left) and Ky(right)")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"})])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            We first investigated how the concentration of Z changes with time when the parameters Kx and Ky are changed independently in Formula xx.1.                            \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("  \n                             Kx, Ky, are the binding coefficients of X and Y to the promoter of gene z. We varied those parameters because actual coefficient values can be controlled by changing the sequence of promoter.                            \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("  \n                            As a result, we found that when the parameter Kx is high, the peak of the pulse become small, and when Ky is high, the sharpness of the peak is lost.                          \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/File:T--Waseda--time-heatmap-add.png",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-7　time heatmap")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            Next, we constructed a heatmap to check how much time (t) of the pulse peak is delayed as we change the parameters Kx and Ky simultaneously. Fig. 2-3-7 is the  results, which show that the lighter the heatmap’s color is, the more the peak time delays. However, when the values of parameters become too large, pulse lost it’s sharpness as the the difference between the maximum value and the steady-state concentration decreases.                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/9/96/T--Waseda--ratio_heatmap.png",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-8 ratio heatmap ")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/9/91/T--Waseda--cross_heatmap.png",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-9 cross heatmap ")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            Furthermore, in order to evaluate the balance of the pulse shape, we calculated the ratio of the peak concentration and the following steady-state concentration for each parameter and presented it in a heat map. This is called the ratio heatmap. (Fig 2-3-8). The darker the color, the smaller the ratio is and the pulse will be in an indistinct form as shown in Fig 2-3-7, Kx=1000.\n                            Fig 2-3-8 shows that the ratio becomes smaller when the Kx value is too high. We evaluated each parameter based on the peak duration in Fig 2-3-7 and the balance of the pulse shape in Fig 2-3-8, respectively. Based on these two indicators, we multiplied the cross heatmap by the value of the time heatmap and the ratio heatmap for each parameter in order to find out which parameter has both time difference and balance, comprehensively (Fig 2-3-9). To make this cross heatmap,\n                             we subtracted a certain constant number of time values from the time heatmap because we have to make the time delay above a certain level. If the time heatmap is negative, it is treated as 0 (fig. 3階層). This is called the time processed heatmap. Then, when multiplying the values of the time processed heatmap and the ratio heatmap, the value of the time processed heatmap was raised to the power of 1.2 and the value of the ratio heatmap was raised to the power of 10 to adjust for the contribution of the respective heatmap values. We selected the optimal parameter for the multi IFFL in the resulting match heatmap as the one with the highest score. Two pulses were generated using the optimal parameter in practice(Fig 2-3-10). Yellow indicates the temporal variation of the concentration of Z1 and green indicates that of Z2. As a result, We have succeeded in creating pulses with a time difference. This demonstrates the feasibility of our SAMURAI system.\n\n                          \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("\n                            Although we sure that you have enjoyed our SAMURAI, however, we also know that real-world implementation requires another effort: business model creation. This is why we've been working on the business model from the early time of the project.                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-10 results")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3"),e._v(" "),o("p",{staticClass:"p_indent"}),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/6/6c/T--Waseda--buisiness_overview.png",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("2-3-11 Overview of business model innovation")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            As well as planning for scientific research, business model creation should learn from previous success.  From Mr. Watanabe’s point of view, a business has a higher probability of success when two of a four points are changed from the existing business success. The kind of change is called business model innovation.                           \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/8/8e/T--Waseda--business_interview.png",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-3-12 Interview of business model")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("    \n                            With with the knowledge we acquired from lectures on business model planning, we created some business models based on our cell-free based iGEM project. Although we created detailed plan which can take advantage of our success in monoterpene syntheses , another interview to experts during the above planning gave us more attractive and valuable business model where a cell-free system is used for substance detection.                          \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box"},[o("h2",{staticClass:"h2_text"},[e._v("Cell-free monoterpene synthesis")])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Background")]),e._v(" "),o("P",{staticClass:"p_indent"},[e._v("\n                              How can we observers distinguish between zombies and samurai?\n                                In our project we propose one unique identification method, though there is also a method using fluorescent protein.\n                                  We propose a method of distinguishing the scents of zombies or samurai.  For this project, we selected monoterpenes,\n                                   especially limonene and sabinene, as the scented substances produced by zombies and samurai.\n                            ")])],1)],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Experiment")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/b/b3/T--Waseda--monoterpene_2-2-1_Parts_and_metabolic_pathways_in_this_experiment_and_a_schematic_diagram_of_the_experiment.png",alt:"Parts_and_metabolic_pathways_in_this_experiment_and_a_schematic_diagram_of_the_experiment"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig. 2-2-1 Parts and metabolic pathways in this experiment and a schematic diagram of the experimen")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("\n                            In this cell-free monoterpene synthesis, we mixed two "),o("i",[e._v("E. coli")]),e._v(" extracts each of which has either first 7 or last 2 enzymes of a pathway from Ac-CoA, which is a major intermediate of cell central metabolism.\n                                Through mevalonate pathway, the former extract one (derived from "),o("i",[e._v("E. coli")]),e._v(" into which pBbA5c-MevT-MBI has been introduced) can provide IPP and DMAPP, which can also be used as intermediates for other important biosynthesis.\n                                  Here we indeed supplemented only glucose and acetate as carbon sources.  We obtained expression system for those seven genes from addgene and have converted this into Biobrick RFC 1000 format by synonymous replacement ("),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580103"}},[e._v(" BBa_K3580103")]),e._v(").\n                                    In order to take advantage of an engineering principle of synthetic biology we provided two biobrick parts ( "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580101"}},[e._v("BBa_K3580101")]),e._v(", "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580102"}},[e._v("BBa_K3580102")]),e._v(" ) for the source for the latter extract.  "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580101"}},[e._v("BBa_K3580101")]),e._v(" has GPP synthase (GPPS) and limonene synthase.\n                                      Although GPP synthase is shared with "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580101"}},[e._v("BBa_K3580101")]),e._v(" , "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part: "}},[e._v("BBa_K3580102")]),e._v(" has sabinene synthase, which has one point mutation in limonene synthase (Srividya Narayanan et al 2015) and a new coding sequence for Parts registry of iGEM (See here for more details on this experiments).\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{ols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Results")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v(" \n                            Similar to a very recent study from Jwett lab (Dudly et al 2019) who mixed 7 extracts, we synthesized limonene using only two extracts.\n                              Although fine-tuning can be possible when a larger number of extracts is prepared, we are sure that entry projects in iGEM should be simple but has engineering principle in order to expand iGEM sucess to an educational tool.\n                                This is why we selected the division into two extracts.                           \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/f/f7/T--Waseda--monoterpene_2-2-2_GCMS_analysis_results_of_limonene_synthesis_system.png",alt:"Fig. 2-2-2 GC/MS analysis results of limonene synthesis system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig. 2-2-2 GC/MS analysis results of limonene synthesis system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v(" \n                            We confirmed limonene synthesis GC/MS analysis with SIM.  In this SIM analysis, ions with four m/z values  characteristic in limonene (68 and 93) and sabinene (77 and 91, 93) were analyzed.   By using authentic limonene standard, we confirmed a retention time for GC and the characteristic limonene SIM signal at the specific m/z values.  At the same retention time with the standard, limonene-specific m/z value (68, 93) ions were detected in the selected ions (The upper right figure of Figure. 2-2-2).\n                              We also draw a GC chart by summation of the signals from the selected ions (The lower left figure of Figure. 2-2-2).  By comparison with a negative control experiment which we omitted the extract containing GPP synthase and limonene synthase, we found clear peak from our limonene synthesis.                           \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/f/f2/T--Waseda--monoterpene_2-2-3_GCMS_analysis_results_of_sabinene_synthesis_system.png",alt:"Fig. 2-2-3 GC/MS analysis results of sabinene synthesis system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig. 2-2-3 GC/MS analysis results of sabinene synthesis system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3"),e._v(" "),o("p",{staticClass:"p_indent"},[e._v(" \n                            By similar GC/MS analysis with SIM, we confirmed the world's first report of sabinene synthesis using cell extracts.  Although elaborative reconstituted system by mixing of purified enzymes has been reported (Srividya Narayanan et al 2015), (Korman Tyler P et al 2017).\n                              Extract-base system with not only simple but engineering principle must provide much contribution to iGEM and DIY biology which as large number of players. Furthermore, extract systems do not require expensive coenzymes (Dudly et al 2019). As well as limonene detection in Fig 2-2-2,\n                               we firstly confirmed sabinene retention time and SIM signal at the m/z values (77, 91, and 93).  From our cell-free production, we then detected sabinene SIM signal and chromatogram peak which cannot find from the negative control.  Also, from the system containing sabinene synthase and GPP synthase,\n                                a peak with the same retention time as the standard limonene product and ions with m/z values characteristic in limonene at that retention time were detected.\n                                 This was consistent with the fact that the paper that referred to the sabinene synthase used in this experiment also reported the synthesis of limonene as a by-product (Srividya Narayanan. et al 2015).                           \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("  \n                            The part "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580102"}},[e._v("BBa_K3580102")]),e._v(" used in our sabinene production is a new part for Parts Registry.   One amino acid mutation was introduced into limonene synthase of "),o("i",[e._v("Mentha spicata")]),e._v(" to create sabinene synthase, by using information reported in a 2015 paper by Srividya and Narayanan et al.                            \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("\n                            We regard our "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580101"}},[e._v("BBa_K3580101")]),e._v(" as an improved version from pre-existing "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3580201"}},[e._v("BBa_K3580201")]),e._v(", which has only limonene synthase.   Our improved part can use the set of IPP DMAPP both of which are provided not only mevalonate pathway used here but the other famous non-mevalonate pathway.  Thus, the set is produced in so many engineered pathways,\n                             including the addgene plasmid used here, as an important intermediate set for bioproduction.  From this view point, we are sure that the simple combination of two enzymes in this part satisfied easy use modularity in synthetic biology than pre-existing "),o("a",{staticClass:"link_style",attrs:{href:"http://parts.igem.org/Part:BBa_K3052001"}},[e._v("BBa_K3052001")]),e._v(".                        \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/f/fa/T--Waseda--monoterpene_2-2-4_Yields_of_monoterpene_per_reaction_solution_in_this_experiment_.png",alt:"Fig. 2-2-4 Yields of monoterpene per reaction solution in this experiment"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig. 2-2-4 Yields of monoterpene per reaction solution in this experiment")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("     \n                            Finally, each of monoterpenes were quantified based on peaks of substance having 93 m/z and each monoterpene standard curves.  Taking advantage of the modularity of the combination of extracts, we confirmed whether the yield of monoterpenes could be changed by changing the mixing ratio of the extract containing the enzyme of the mevalonate pathway and the extract containing GPP synthase and monoterpene synthase.\n                              As a result, changes in the yield of monoterpenes due to the mixing ratio of the extracts were observed (Figure. 2-2-4).  The best yield of limonene per reaction solution with limonene synthase contained system was 0.73 µM, and the best yield of sabinene per reaction solution with sabinene synthase contained system was 4.6 µM.                        \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Reference")]),e._v(" "),o("p",[e._v("    \n\n                              (1) Alonso-Gutierrez, J. et al.  (2013). Metabolic engineering of "),o("i",[e._v("Escherichia coli")]),e._v(" for limonene and perillyl alcohol production. Metabolic engineering, 19, 33-41. "),o("br"),e._v("\n                              (2) Dudley, Q. M. et al. (2019). Cell-free biosynthesis of limonene using enzyme-enriched "),o("i",[e._v("Escherichia coli")]),e._v(" lysates. Synthetic Biology, 4(1), ysz003. "),o("br"),e._v("\n                              (3) Srividya, N. et al. (2015). Functional analysis of (4S)-limonene synthase mutants reveals determinants of catalytic outcome in a model monoterpene synthase. Proceedings of the National Academy of Sciences, 112(11), 3332-3337."),o("br"),e._v("\n                              (4) Korman, T. P. et al. (2017). A synthetic biochemistry platform for cell free production of monoterpenes from glucose. Nature communications, 8(1), 1-8."),o("br"),e._v('\n                              (5) Mass Spectrometry Data Center, William E. Wallace, "Mass Spectra" in NIST Chemistry WebBook, NIST Standard Reference Database Number 69, Eds. P.J. Linstrom and W.G. Mallard, National Institute of Standards and Technology, Gaithersburg MD, 20899, https://doi.org/10.18434/T4D303, (retrieved October 22, 2020).'),o("br")])])],1)],1),e._v(" "),o("v-row",{attrs:{id:"Racemase"}},[o("v-row",{staticClass:"row_max-box"},[o("h2",{staticClass:"h2_text"},[e._v("Racemase ~War In The Mirror~")])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Introduction")]),e._v(" "),o("P",{staticClass:"p_indent"},[e._v("\n                              In the Zombie vs. Samurai storyline, we envisioned a scenario in which one side deprives their opponent of the food they need.  \n                              A battle between two enantiomers, that is to say, one side has D amino acid and the other has L amino acid derived food as their food source, and both sides converts their opponent’s amino acid and deprives the of their food (Fig.2-4-1). Although there is not yet a translation and transcription system that efficiently incorporates D amino acids, it is known that the originally natural enzyme chemically synthesized with D amino acids would be active.\n\n                            ")])],1)],1)],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/c/c7/T--Waseda--racemase_war_1st.png",alt:"Fig.2-4-1 Images of War In The Mirror in the Zombie vs Samurai storyline"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-4-1 Images of War In The Mirror in the Zombie vs Samurai storyline")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            Although there is not yet a translation and transcription system that efficiently encorporates D amino acids, it is known that the natural enzyme chemically synthesized with D amino acids would be active.A protein made up of only d-amino acids act on substrates with opposite chirality, as demonstrated using the HIV protease [Milton et.al 1992].\n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("     \n                            Besides, practical usage of D-amino acids is advancing in the field of Biotechnology today.  Peptides composed of D-amino acids are not susceptible to degradation by protease in L-body organisms, thus are highly stable [Tugyi et.al 2005][Garton et.al 2018].\n                             Consequently, researchers are seeking a way to make the target of D-amino acid Peptide Aptamers(A type of antibody) L-body organisms’ protein [Oberthür et.al 2015][ Majier et.al 2016][ Schumacher et.al 1996][ Funke et.al 2009]. Currently, target proteins are synthesized with D-bodies, and L-body peptide aptamers that bonds to the protein are artificially evolutionized. When protein is expressed with the obtained amino acid sequence with D-amino acids, the binding site would consist of L-bodies [Oberthür et.al 2015][ Majier et.al 2016][ Schumacher et.al 1996][ Funke et.al 2009].                         \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("\n                            Such storyline can be applied to our real experimental system by adopting racemase, an enzyme which catalyzes interconversion of D amino acid and L amino acid.  Racemase catalyzes interconversion between L-amino acid and D-amino acid. There are many kinds of racemases and each corresponds to respective amino acid. This year, we focused especially on alanine racemase. Alanine racemase (AR) is a fold type III racemase enzyme catalyze the conversion of L-alanine to D-alanine[Walish 1989]. This enzyme provides D-alanine consuming L-alanine and using pyridoxal 5- phosphate (PLP) as a cofactor. AR is unique to prokaryotes and plays an important role in a biosynthesis[Walish 1989] [Azam et.al 2016]. Since D-alanine is used as a necessary component of the peptidoglycan layer of bacterial cell walls, lack of alanine racemase can result in the inhibition of growth of prokaryotes [Azam et.al 2016].\n                             As it is generally absent in higher eukaryotes such as human but is ubiquitous throughout bacteria. Therefore, alanine racemase is recognized as an attractive target for antibacterial drug development [Azam et.al 2016].                            \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Summary & Experiment")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v(" \n                            In this experiment, alanine racemase (AR) gene carried on our new part (BBa_K3580200) on a T5 promoter-controlled pCA24N vector was transformed into the BL21(DE3) star strain. After expression induction by IPTG, the BL21 cells were harvested by centrifugation, then sonicated and His-Tag purification was performed (Fig.2-4-2).                           \n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/b/b8/T--Waseda--racemase_expression.png",alt:"Fig.2-4-2 Purification of alanine racemase expressed in E. Coli"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-4-2 Purification of alanine racemase expressed in "),o("i",[e._v("E. Coli")])])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("  \n                            To assess the activity of purified racemase, fluorescence values of GFPS1 synthesized by a combination of GFPS1 plasmid, L-alanine, D-alanine, and purified AR were measured in the Pure System, a reconstituted CFPS that contains no enzymes other than the translation system [Shimizu et.al 2001] (Fig.2-4-3).\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/a/a1/T--Waseda--racemase_puresystem.png",alt:"Fig.2-4-3 Cell-free protein synthesis in the PURE system"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-4-3 Cell-free protein synthesis in the PURE system")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("     \n                            The natural translation system is difficult to take up D-amino acids.[Kuncha 2019].19 amino acids other than alanine are L-bodies, so if only alanine is D-body, even if the sequence is correct, it has no activity as an enzyme (Fig.2-4-4).                         \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/2/2a/T--Waseda--racemase_pure_alanine.png",alt:"Fig.2-4-4 Cell-free protein synthesis under each alanine conditions"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-4-4 Cell-free protein synthesis under each alanine conditions")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v('   \n                            Since GFP is composed of L-amino acids, when GFPS1 takes up L-alanine, the system would give off fluorescent effect, and when D-alanine and AR are added, the racemized L-alanine can be used for translation and the fluorescence value was expected to be restored(Fig.2-4-5). The evaluation of the effect of amino acids on the synthesis of a reporter protein is the most appropriate methodology for this scenario as a "food source".\n                             As PURE frex does not contain any extra metabolic enzymes other than those involved in transcription and translation, we would be able to perform the assays without having to control the metabolic system, which could plague the assays with extract based CFPS.                           \n                          ')])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/e/e5/T--Waseda--racemase_pure_racemase.png",alt:"Fig.2-4-5 Cell-free protein synthesis using racemization of alanine by racemase"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-4-5 Cell-free protein synthesis using racemization of alanine by racemase")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("\n                            We decided to assay AR by adopting PURE frex donated by GeneFrontier, Inc. After identifying L-alanine contamination problems of commercial D-alanine, the assay with AR was confirmed by the combination of each alanine and racemase as well as the assay with extracts. The results of this experiment are as follows (Fig.2-4-6).                              \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_mid_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/3/36/T--Waseda--racemase_201023_bar.png",alt:"Fig.2-4-6 Effect of L-alanine substrate repletion for translation by racemase"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig.2-4-6 Effect of L-alanine substrate repletion for translation by racemase")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("\n                            The results demonstrated that GFP could not be synthesized by D-alanine alone, but L-alanine produced by racemase-mediated racemization of D-alanine and L-alanine could be used for translation, creating a situation in which fluorescence was restored by the synthesized GFP.                              \n                            In other words, we succeeded in creating a situation in which the opponent’s food was converted into food for themselves in the Zombie vs. Samurai scenario.\n                          ")]),e._v(" "),o("p",{staticClass:"p_indent"},[e._v("     \n                            We also used a cell-free transcription-translation system using cell extracts prior to our experiments with the pure system, but we were unable to obtain clear results due to the production of alanine by the metabolic enzymes in the extracts.\n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Reference")]),e._v(" "),o("p",[e._v("\n                            (1) [Milton et.al 1992] Milton, R., Milton, S., & Kent, S. (1992). Total chemical synthesis of a D-enzyme: The enantiomers of HIV-1 protease show reciprocal chiral substrate specificity [corrected]. Science, 256(5062), 1445-1448. doi:10.1126/science.1604320 "),o("br"),e._v("\n                            (2) [Tugyi et.al 2005] Tugyi, R., Uray, K., Iván, D., Fellinger, E., Perkins, A., & Hudecz, F. (2005). Partial d-amino acid substitution: Improved enzymatic stability and preserved ab recognition of a MUC2 epitope peptide. Proceedings of the National Academy of Sciences, 102(2), 413-418. doi:10.1073/pnas.0407677102"),o("br"),e._v("\n                            (3) [Garton et.al 2018] Garton, M., Nim, S., Stone, T. A., Wang, K. E., Deber, C. M., & Kim, P. M. (2018). Method to generate highly stable D-amino acid analogs of bioactive helical peptides using a mirror image of the entire PDB. Proceedings of the National Academy of Sciences, 115(7), 1505-1510. doi:10.1073/pnas.1711837115"),o("br"),e._v("\n                            (4) [Oberthür et.al 2015] Oberthür, D., Achenbach, J., Gabdulkhakov, A., Buchner, K., Maasch, C., Falke, S., Rehders, D., Klussmann, S., Betzel, C. (2015). Crystal structure of a mirror-image L-RNA aptamer (spiegelmer) in complex with the natural L-protein target CCL2. Nature Communications, 6(1), 6923. doi:10.1038/ncomms7923"),o("br"),e._v("\n                            (5) [Majier et.al 2016] Maier, K., & Levy, M. (2016). From selection hits to clinical leads: Progress in aptamer discovery. Molecular Therapy.Methods & Clinical Development, 5, 16014. doi:10.1038/mtm.2016.14"),o("br"),e._v("\n                            (6) [Schumacher et.al 1996] Schumacher, T. N. M., Mayr, L. M., Minor, D. L., Milhollen, M. A., Burgess, M. W., & Kim, P. S. (1996). Identification of d-peptide ligands through mirror-image phage display. Science, 271(5257), 1854-1857. doi:10.1126/science.271.5257.1854"),o("br"),e._v('\n                            (7) [Funke et.al 2009] Funke, S. A., & Willbold, D. (2009). Mirror image phage display—a method to generate d-peptideligands for use in diagnostic or therapeutical applications. Mol.BioSyst., 5(8), 783-786. doi:10.1039/B904138A"'),o("br"),e._v("\n                            (8) [Walish 1989] Walsh, C. T. (1989). Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. The Journal of Biological Chemistry, 264(5), 2393-2396."),o("br"),e._v("\n                            (8) [Azam et.al 2016] Azam, M. A., & Jayaram, U. (2016). Inhibitors of alanine racemase enzyme: A review. Journal of Enzyme Inhibition and Medicinal Chemistry, 31(4), 517-526. doi:10.3109/14756366.2015.1050010"),o("br"),e._v("\n                            (9) [Shimizu et.al 2001] Shimizu, Y., Inoue, A., Tomari, Y., Suzuki, T., Yokogawa, T., Nishikawa, K., & Ueda, T. (2001). Cell-free translation reconstituted with purified components. Nature Biotechnology, 19(8), 751-755. doi:10.1038/90802"),o("br"),e._v("\n                            (10) [Kuncha et.al 2019] Kuncha, S. K., Kruparani, S. P., & Sankaranarayanan, R. (2019). Chiral checkpoints during protein biosynthesis. The Journal of Biological Chemistry, 294(45), 16535-16548. doi:10.1074/jbc.REV119.008166"),o("br")])])],1),e._v(" "),o("v-row",{directives:[{name:"scroll",rawName:"v-scroll",value:e.pushScrollY,expression:"pushScrollY"}],attrs:{id:"Education"}},[o("v-row",{staticClass:"row_max-box"},[o("h2",{staticClass:"h2_text"},[e._v("Education")])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3",[e._v("Overview")]),e._v(" "),o("P",{staticClass:"p_indent"},[e._v("\n                              At initial stage of our iGEM activity, we realized that not only those with little scientific knowledge but also students who aspired to be scientists didn’t recognize synthetic biology. As iGEMers, who mainly work with synthetic biology, we felt the need to spread the knowledge of its potential to produce world-changing results in both engineering and physiological sciences.  This year, we’re attracting attention with an interesting story whose subject is gene circuit.  We tried to spread toggle switch, which was core of the circuit and starting point of synthetic biology.\n                            ")])],1)],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_mid_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/2/21/T--Waseda--Education_Application_student.png",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("Fig 2-6-1 EducationApplication student")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("      \n                            We decided to develop a smartphone application (Fig. 2-6-1) for better understanding of mathematical modeling of synthetic biology for the synthetic biology beginners. Mathematical modeling is fundamental in synthetic biology, for engineering principle requires such modeling.  We fortunately had a chance to lecture Waseda University High School students. The lecture was held through a widely used video communication service, zoom.\n                                                  \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"https://2020.igem.org/wiki/images/f/f0/T--Waseda--Education_Application_Explanation.png",alt:"fig 2.7 toggle switch simulator"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"},[e._v("fig 2-6-2 toggle switch simulator")])])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("p",{staticClass:"p_indent"},[e._v("   \n\n                            The application shows how genetic circuit works, especially about toggle switch. Rates of change in repressor concentrations move according to parameter values(⑤).  Resulting in visual information such as graph or color transitions(③,④,⑥).  Each student can move the parameter values(⑤),  watching the feedback, and learn how each parameter affects the result.   \n                            By operating the app hands-on, we received positive comments from the students.   Some said they felt fresh about the way they recognized organisms by using mathematical model, and others said they found utility in this method.\n                            Afterwards, this smartphone application was incorporated in a university class because of its usefulness.\n                                                      \n                          ")])])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"})])]),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("v-col",[o("h3"),e._v(" "),o("p",{staticClass:"p_indent"}),e._v(" "),o("p",{staticClass:"p_indent"})])],1),e._v(" "),o("v-row",{staticClass:"row_max-box",attrs:{cols:"12",sm:"12",xs:"12",md:"12",lg:"7",xl:"7"}},[o("figure",{staticClass:"fig_full_width"},[o("img",{staticClass:"image_max",attrs:{src:"",alt:"Logo"}}),e._v(" "),o("figcaption",{staticClass:"fig_title"})])])],1),e._v(" "),o("v-template",{directives:[{name:"scroll",rawName:"v-scroll",value:e.pushScrollY,expression:"pushScrollY"}],attrs:{id:"section_06"}},[o("v-col",{staticClass:"last_message"})],1)],1)],1)],1)],1)}),[],!1,null,"634cd977",null);t.default=component.exports;c()(component,{VCol:m.a,VContainer:d.a,VRow:h.a}),_()(component,{Scroll:v.b})}}]);